inc/options/class-wpseo-option-ms.php

```diff
@@ -83,49 +83,51 @@ public static function get_instance() {
-			"{$allow_prefix}disableadvanced_meta"           => true,
-			"{$allow_prefix}ryte_indexability"              => true,
-			"{$allow_prefix}content_analysis_active"        => true,
-			"{$allow_prefix}keyword_analysis_active"        => true,
-			"{$allow_prefix}enable_admin_bar_menu"          => true,
-			"{$allow_prefix}enable_cornerstone_content"     => true,
-			"{$allow_prefix}enable_xml_sitemap"             => true,
-			"{$allow_prefix}enable_text_link_counter"       => true,
-			"{$allow_prefix}enable_headless_rest_endpoints" => true,
-			"{$allow_prefix}enable_metabox_insights"        => true,
-			"{$allow_prefix}enable_link_suggestions"        => true,
-			"{$allow_prefix}tracking"                       => true,
-			"{$allow_prefix}enable_enhanced_slack_sharing"  => true,
-			"{$allow_prefix}semrush_integration_active"     => true,
-			"{$allow_prefix}zapier_integration_active"      => true,
-			"{$allow_prefix}wincher_integration_active"     => false,
-			"{$allow_prefix}remove_feed_global"             => true,
-			"{$allow_prefix}remove_feed_global_comments"    => true,
-			"{$allow_prefix}remove_feed_post_comments"      => true,
-			"{$allow_prefix}enable_index_now"               => true,
-			"{$allow_prefix}remove_feed_authors"            => true,
-			"{$allow_prefix}remove_feed_categories"         => true,
-			"{$allow_prefix}remove_feed_tags"               => true,
-			"{$allow_prefix}remove_feed_custom_taxonomies"  => true,
-			"{$allow_prefix}remove_feed_post_types"         => true,
-			"{$allow_prefix}remove_feed_search"             => true,
-			"{$allow_prefix}remove_atom_rdf_feeds"          => true,
-			"{$allow_prefix}remove_shortlinks"              => true,
-			"{$allow_prefix}remove_rest_api_links"          => true,
-			"{$allow_prefix}remove_rsd_wlw_links"           => true,
-			"{$allow_prefix}remove_oembed_links"            => true,
-			"{$allow_prefix}remove_generator"               => true,
-			"{$allow_prefix}remove_emoji_scripts"           => true,
-			"{$allow_prefix}remove_powered_by_header"       => true,
-			"{$allow_prefix}remove_pingback_header"         => true,
-			"{$allow_prefix}clean_campaign_tracking_urls"   => true,
-			"{$allow_prefix}clean_permalinks"               => true,
-			"{$allow_prefix}search_cleanup"                 => true,
-			"{$allow_prefix}search_cleanup_emoji"           => true,
-			"{$allow_prefix}search_cleanup_patterns"        => true,
-			"{$allow_prefix}wordproof_integration_active"   => false,
+			'access'                                            => 'admin',
+			'defaultblog'                                       => '', // Numeric blog ID or empty.
+			"{$allow_prefix}disableadvanced_meta"               => true,
+			"{$allow_prefix}ryte_indexability"                  => true,
+			"{$allow_prefix}content_analysis_active"            => true,
+			"{$allow_prefix}keyword_analysis_active"            => true,
+			"{$allow_prefix}inclusive_language_analysis_active" => true,
```

Comment:
Probably this has to be `false`.

Reply:
Also, there's apparently a merge conflict in this file 😮

Reply:
> Also, there's apparently a merge conflict in this file 😮

That's a known issue. See [this thread on Slack](https://yoast.slack.com/archives/C5SUKMF2T/p1658738092238229). I will check if I can solve it by merging the cherrypick branch into this branch.

Reply:
I merged the cherrypick branch into this branch, which solves the merge conflict. It did add some extra commits from the cherrypick branch that are not (yet) in trunk though.